news/2021-09-26-project-loved-september-2021.md

```diff
@@ -0,0 +1,385 @@
+
+[Arise - 7K Endurance and technical training pack](https://osu.ppy.sh/beatmapsets/699875#mania) by [Arise](https://osu.ppy.sh/users/5127544)\
+144 – 250 BPM, 4:07 | 4.97★ – 7.09★\
+The \[Mirror went\] difficulty is *not* being nominated for Loved.\
+*description written by [\_underjoy](https://osu.ppy.sh/users/2235750)*
+
+Most of 7K players probably have played a chart from this pack at least once in their o!m career. Arise's *7K Endurance and technical training pack* (also known as Xcaliboor, especially in the 4K community) is a seasoned classic gallery of relatively standard chordstream charts with some LN sections here and there. The choice of difficulties ranges from averagely hard (like Midnight lighting bolt and Sister's noise) to very challenging (Aim burst and Love is war). Although the pack's name mentions technical patterning, its theme is focused on endurance, and there's a tiny technicality in the charts apart from some sparse LN sections. Now a word of warning - patterning of chordstreams used is very basic and doesn't stand out for the most part. The quality of some charts isn't stellar, but the pack never was about following the music as closely as possible. As this is the Endurance pack, its main focus is stamina, and playing these charts is an excellent way to improve the 7K muscles and get used to chordstreaming on many different levels. We hope you will warmly welcome Arise's pack as there is a lot of content to set a score on!
+
+[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/1220647.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[osu!mania 7K Dan Course - LN Dan Phase III](https://osu.ppy.sh/beatmapsets/1220647#mania) by [Jinjin](https://osu.ppy.sh/users/3360737), [\_Kobii](https://osu.ppy.sh/users/6209713), [Kim_GodSSI](https://osu.ppy.sh/users/2218047), [Pengdoll](https://osu.ppy.sh/users/6392061) and [Toaph Daddy](https://osu.ppy.sh/users/7616811)\
+167 – 198 BPM, 8:13 | 9.79★, 10.48★, 11.76★, 12.29★, 13.26★\
+*description written by [\_underjoy](https://osu.ppy.sh/users/2235750)*
+
+There it is, the final exam of 7K LN abilities and the last part of revamped Jinjin Dans that go up into the Loved voting - *LN Dan Phase III*! There are 5 Dans in this pack: 9th, 10th, Gamma, Azimuth, and Zenith. Since all of you probably know what they are about, I'll keep this short. Dans are sets of 4 songs played consecutively with small breaks inbetween. Each piece is focused on a different aspect of gameplay; in the case of the LN Dans, there are general, tech, inverse, and LN release maps. Passing the course with 95% or more counts as a clear. If you want to know more, be sure to check the mapset's description. These courses are a way to track your progress in the LN department and act as a standard of difficulty for other maps to compare to. Be sure to vote for the last remaining puzzle in the 7K Jinjin Dan set, and good luck with clearing some of the hardest 7K LN courses osu!mania has to offer!
```

Comment:
```suggestion
There it is, the final exam of 7K LN abilities and the last part of revamped Jinjin Dans that go up into the Loved voting — *LN Dan Phase III*! There are 5 Dans in this pack: 9th, 10th, Gamma, Azimuth, and Zenith. Since all of you probably know what they are about, I'll keep this short. Dans are sets of 4 songs played consecutively with small breaks inbetween. Each piece is focused on a different aspect of gameplay; in the case of the LN Dans, there are general, tech, inverse, and LN release maps. Passing the course with 95% or more counts as a clear. If you want to know more, be sure to check the mapset's description. These courses are a way to track your progress in the LN department and act as a standard of difficulty for other maps to compare to. Be sure to vote for the last remaining puzzle in the 7K Jinjin Dan set, and good luck with clearing some of the hardest 7K LN courses osu!mania has to offer!
```